fenzo-core/src/main/java/com/netflix/fenzo/queues/tiered/Tier.java

```diff
@@ -270,6 +271,46 @@ private String getSortedListString() {
         return b.toString();
     }
 
+    private String getTierCapacityString() {
+        StringBuilder sb = new StringBuilder();
+        if (tierResources != null) {
+            sb.append("Tier ").append(tierNumber).append(" Total Capacity: { ");
+            sb.append("cpu: ").append(tierResources.getCores());
+            sb.append(", memory: ").append(tierResources.getMemory());
+            sb.append(", disk: ").append(tierResources.getDisk());
+            sb.append(", networkMbps: ").append(tierResources.getNetworkMbps()).append(" }");
+        }
+
+        if (effectiveUsedResources != null) {
+            sb.append("\nTier ").append(tierNumber).append(" Used Capacity: { ");
+            sb.append("cpu: ").append(effectiveUsedResources.getCores());
+            sb.append(", memory: ").append(effectiveUsedResources.getMemory());
+            sb.append(", disk: ").append(effectiveUsedResources.getDisk());
+            sb.append(", networkMbps: ").append(effectiveUsedResources.getNetworkMbps()).append(" }");
+        }
+
+        if (remainingResources != null) {
+            sb.append("\nTier ").append(tierNumber).append(" Unused Capacity: { ");
+            sb.append("cpu: ").append(remainingResources.getCores());
+            sb.append(", memory: ").append(remainingResources.getMemory());
+            sb.append(", disk: ").append(remainingResources.getDisk());
+            sb.append(", networkMbps: ").append(remainingResources.getNetworkMbps()).append(" }");
+        }
+        return sb.toString();
+    }
+
+    public String getBucketCapacityString(QueueBucket bucket) {
```

Comment:
It would be useful to see the current bucket consumption. Bucket guarantees only show how much we have allocated.